app/Console/Commands/GetWpProducts.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\Dashboard;
+use App\Models\WpProducts;
+use App\Models\WpProductVariants;
+use Automattic\WooCommerce\Client;
+use Carbon\Carbon;
+
+class GetWpProducts extends Command
+{
+
+    public $client;
+    protected $signature = 'products:update_wp_product {time_report?} {store?}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Update Products';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $storeRequest = $this->argument('store') ?? '';
+        $stores = Dashboard::getWooStoresList();
+        if ($storeRequest) {
+            $stores = array($storeRequest);
+        }
+        $this->info("Cron Job $storeRequest Update Products  running at " . now());
+
+        foreach ($stores as $store) {
+            $wooConfig = Dashboard::getWoocommerceConfig($store);
+          
+            $domain = $wooConfig['domain'];
+            $consumerKey = $wooConfig['consumerKey'];
+            $consumerSecret = $wooConfig['consumerSecret'];
+            $apiVersion = $wooConfig['apiVersion'];
+            $dateTimeZone = $wooConfig['dateTimeZone'];
+
+            $dateTime = new \DateTime("now", $dateTimeZone);
+            $dateTime->modify('-1 day');
+            $updatedAtMin = $dateTime->format('Y-m-d');
+
+            $timeReport = $this->argument('time_report') ?? '';
+            if ($timeReport == 'all') {
+                $updatedAtMin = '1900-01-01';
+            }
+
```

Comment:
Xóa code này nếu không dùng đến giúp anh nhé:

$dateTime = new \DateTime("now", $dateTimeZone);
$dateTime->modify('-1 day');
$updatedAtMin = $dateTime->format('Y-m-d');

$timeReport = $this->argument('time_report') ?? '';
if ($timeReport == 'all') {
    $updatedAtMin = '1900-01-01';
}